src/app/server/CommissioningWindowManager.cpp

```diff
@@ -184,13 +184,21 @@ CHIP_ERROR CommissioningWindowManager::OpenCommissioningWindow()
     }
     else
     {
-        uint32_t pinCode;
-        ReturnErrorOnFailure(DeviceLayer::ConfigurationMgr().GetSetupPinCode(pinCode));
-
-        ReturnErrorOnFailure(mPairingSession.WaitForPairing(
-            pinCode, kSpake2p_Iteration_Count,
-            ByteSpan(reinterpret_cast<const uint8_t *>(kSpake2pKeyExchangeSalt), strlen(kSpake2pKeyExchangeSalt)), keyID,
-            Optional<ReliableMessageProtocolConfig>::Value(gDefaultMRPConfig), this));
+        uint32_t iterationCount;
+        uint8_t salt[kPBKDFMaximumSaltLen];
+        size_t saltLen;
+        PASEVerifier verifier;
+        size_t verifierLen;
```

Comment:
Please initialize all of these to guard against ConfigurationMgr() impls that do not return error, but return invalid data

Reply:
done